src/main/java/net/william278/velocitab/player/TabPlayer.java

```diff
@@ -98,11 +98,20 @@ public CompletableFuture<Component> getDisplayName(@NotNull Velocitab plugin) {
 
     }
 
+    @NotNull
+    public CompletableFuture<String> getNametag(@NotNull Velocitab plugin) {
+        final String serverGroup = plugin.getSettings().getServerGroup(getServerName());
+        return Placeholder.replace(plugin.getSettings().getNametag(serverGroup), plugin, this)
+                .thenApply(formatted -> plugin.getFormatter().format(formatted)
+                        .replace(player.getUsername(), "%username%"));
```

Comment:
We should probably do some error checking if %username% is not present to warn users of issues doing this